### PR TITLE
updated three.js to r123

### PR DESCRIPTION
**Pull request by elalish**
@mrdoob I appreciate the `.inverse()` change!

### CODE REVIEW

**Comment by mrdoob** [comment]
> @mrdoob I appreciate the `.inverse()` change!

🤓!

File: packages/model-viewer/src/three-components/ARRenderer.ts [hunk 1]
```diff
@@ -529,12 +529,11 @@ export class ARRenderer extends EventDispatcher {
       origin.sub(direction.multiplyScalar(model.idealCameraDistance));
       const ray = new Ray(origin, direction.normalize());
 
-      const modelToWorld = this.initialModelToWorld;
+      const modelToWorld = this.initialModelToWorld!;
```
Comment (mrdoob) [inline packages/model-viewer/src/three-components/ARRenderer.ts:532]: Does adding a `!` a the end clone the object or something?

Comment (elalish) [reply packages/model-viewer/src/three-components/ARRenderer.ts:532]: No, that's just telling TS don't bug me about this maybe being null, I know it's really there.

Comment (mrdoob) [reply packages/model-viewer/src/three-components/ARRenderer.ts:532]: Right, something about the new code seems fishy. Remind me the next time we sync.

**Pull request merged**